addons/mrp/models/stock_rule.py

```diff
@@ -61,6 +61,7 @@ def _run_manufacture(self, procurements):
                     ('picking_type_id', '=', rule.picking_type_id.id),
                     ('company_id', '=', procurement.company_id.id),
                     ('user_id', '=', False),
+                    ('location_dest_id', '=', procurement.location_id.id)
```

Comment:
I would use `child_of` instead of `=`

Reply:
Thanks for the review, I also adapted the test for this use case ;)